src/main/java/racingcar/Application.java

```diff
@@ -1,7 +1,90 @@
 package racingcar;
 
+import camp.nextstep.edu.missionutils.Console;
+import camp.nextstep.edu.missionutils.Randoms;
+
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStreamReader;
+import java.util.Arrays;
+import java.util.stream.Collectors;
+
 public class Application {
     public static void main(String[] args) {
         // TODO: 프로그램 구현
+
+        System.out.println("경주할 자동차 이름을 입력하세요.(이름은 쉼표(,) 기준으로 구분)");
+        String carNames = Console.readLine();
```

Comment:
`Console.readLine()`을 잘 사용하셨는데 자주 사용하는 메소드이죠?
`import static camp.nextstep.edu.missionutils.Console.readLine;`처럼 
static import를 사용한다면 아래처럼 사용할 수도 있답니다😮 
```suggestion
        String carNames = readLine();
```